pkg/cluster/internal/create/nodes.go

```diff
@@ -88,6 +94,18 @@ func provisionNodes(
 	return nil
 }
 
+func createNetwork(status *logutil.Status, clusterName string) error {
+	defer status.End(false)
+
+	status.Start("Creating bridge network 🌉")
+	if err := docker.CreateNetwork(clusterName); err != nil {
```

Comment:
we still need to handle if the network already exists.

Reply:
Yes, I am doing this and will submit it immediately.

Reply:
> we still need to handle if the network already exists.

I am using `docker.IsNetworkExist` to check if the network is exist.  https://github.com/kubernetes-sigs/kind/pull/484/files#diff-62943c16bb4ee62933f24631e2d3d507R84